app/validators/expiry_date_validator.rb

```diff
@@ -0,0 +1,10 @@
+class ExpiryDateValidator < ActiveModel::Validator
+  def validate(record)
+    if !record.expires_at.nil? && record.expires_at <= DateTime.now
```

Comment:
You should always prefer `Date`, `DateTime`, `Time` `#current` over `#now`. You should read the docs to understand why.

Be careful when writing logic statements. If you can avoid double negatives, you should. 
`!record.expires_at.nil?` can just be `record.expires_at` since in ruby objects are truthy.